packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -78,6 +102,27 @@ export class PreferencesController extends BaseController<
       },
       showTestNetworks: false,
       isIpfsGatewayEnabled: true,
+      showIncomingTransactions: {
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.MAINNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.GOERLI]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.BSC]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.BSC_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.OPTIMISM]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.OPTIMISM_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.POLYGON]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.POLYGON_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.AVALANCHE]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.AVALANCHE_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.FANTOM]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.FANTOM_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.SEPOLIA]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.LINEA_GOERLI]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.LINEA_MAINNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.MOONBEAM]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.MOONBEAM_TESTNET]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.MOONRIVER]: true,
+        [ETHERSCAN_SUPPORTED_CHAIN_IDS.GNOSIS]: true,
+      },
```

Comment:
This can be derived from `ETHERSCAN_SUPPORTED_CHAIN_IDS` as well.

```suggestion
      showIncomingTransactions: Object.values(
        ETHERSCAN_SUPPORTED_CHAIN_IDS,
      ).reduce((acc, curr) => {
        acc[curr] = true;
        return acc;
      }, {} as { [chainId in EtherscanSupportedHexChainId]: boolean }),
```